.circleci/config.yml

```diff
@@ -196,29 +196,35 @@ workflows:
   workflow:
     jobs:
       - build-job:
-          name: "Xcode_11.4_iOS_13.4"
-          xcode: "11.4.0"
-          iOS: "13.4"
+          name: "Xcode_11.5_iOS_13.5"
+          xcode: "11.5.0"
+          iOS: "13.5"
+          device: "iPhone 11 Pro"
+      - build-job:
+          name: "Xcode_11.4.1_iOS_13.4.1"
+          xcode: "11.4.1"
+          iOS: "13.4.1"
           device: "iPhone 8 Plus"
       - build-job:
-          name: "Xcode_10.3_iOS_12.1"
-          xcode: "10.3.0"
-          iOS: "12.1"
+          name: "Xcode_11.4.1_iOS_12.2"
+          xcode: "11.4.1"
+          iOS: "12.2"
           codecoverage: true
       - build-job:
-          name: "Xcode_10.2_iOS_10.3.1"
-          xcode: "10.2.1"
+          name: "Xcode_11.4.1_iOS_10.3.1"
+          xcode: "11.4.1"
```

Comment:
We need to continue to test against iOS 10._x_ somehow, because the SDK still supports iOS 10._x_. On the other hand, it’s fine to drop Xcode 10 support, but we should update the readme to make that clear.

Reply:
I've added step to test against iOS `10.3.1`. Do you think that's enough? Also another note is that since properties like `courseAccuracy` were introduced only in iOS `13.4` minimum usable version of Xcode is `11.4.1`. Please let me know if that's okay.